playbook.md

```diff
@@ -783,9 +783,12 @@ Most developers have maintenance responsibilities, which they do during [ticket
 We do our very best to work to a [sustainable pace](#sustainable-pace). But sometimes, when we're approaching a firm deadline or a launch, or a client is having an emergency, we work longer hours than normal. From time to time, there's an emergency that means we have to work during unsociable hours to solve the problem. Neither of these happen very often, but they are a normal part of life at dxw.
 
 ### Remuneration
-dxw will institute a cost of living rise each year from April 2017. The percentage will be set by the CPI Index (https://www.ons.gov.uk/economy/inflationandpriceindices) as of 1st April each year and you will receive your raise in the April payroll.
+dxw will institute a cost of living rise each year on 1st April. The percentage will be set by the CPI Index (https://www.ons.gov.uk/economy/inflationandpriceindices) as of 1st April each year. 
 
-For any other salary queries, please speak to Harry.
+- If you have passed your probation, you will receive your raise in the April payroll. 
+- For those team members who joined prior to 1st April of that year, but haven't yet passed probation, you will receive your raise in the month you successfully complete it and it will be the same percentage as of 1st April CPI that year. 
```

Comment:
Could you add something to explain why that's the case, please? Inflation doesn't care whether we're in probation or not :) so it doesn't seem intuitive.

Reply:
It is not unusual to delay benefits until probation is passed. I hope that helps. Keen to get this commit closed.

Reply:
Can I resolve this conversation and get this merged please.